modules/server/searchbroker/main.tf

```diff
@@ -24,7 +24,7 @@ data "template_file" "searchbroker_ui_service" {
     mdr_api_url     = terraform.workspace == "default" ? "https://mdr.germanbiobanknode.de/v3/api/mdr" : format("https://mdr.%s.bbmri.de/v3/api/mdr", terraform.workspace)
     mdr_namespace   = "mdr16"
     mdr_language    = "en"
-    mdr_mapping     = "<dataElementGroup nameOfEnum=\"DONOR\">urn:mdr16:dataelementgroup:5:1</dataElementGroup><dataElementGroup nameOfEnum=\"SAMPLE_CONTEXT\">urn:mdr16:dataelementgroup:4:1</dataElementGroup><dataElementGroup nameOfEnum=\"SAMPLE\">urn:mdr16:dataelementgroup:3:1</dataElementGroup><dataElementGroup nameOfEnum=\"EVENT\">urn:mdr16:dataelementgroup:6:1</dataElementGroup>"
+    mdr_mapping     = "<dataElementGroup nameOfEnum=\"DONOR\">urn:mdr16:dataelementgroup:5:1</dataElementGroup><dataElementGroup nameOfEnum=\"SAMPLE\">urn:mdr16:dataelementgroup:3:1</dataElementGroup><dataElementGroup nameOfEnum=\"EVENT\">urn:mdr16:dataelementgroup:6:1</dataElementGroup>"
```

Comment:
Ich bin mit der Syntax nicht ganz sicher, Kann es sein, dass wir innerhalb des Strings statt \" die Schreibweise \\\" verwenden müssen?

Siehe https://www.terraform.io/docs/configuration-0-11/interpolation.html Abschnitt "Built-in Functions"

Reply:
Thanks for reviewing. We use Terraform 0.12 and use \" to get an " corresponding to https://www.terraform.io/docs/configuration/expressions.html#string-literals
I verified this on the server: 
![image](https://user-images.githubusercontent.com/44226714/72822184-f786d080-3c71-11ea-90da-f537a0475d84.png)

Reply:
Thanks for verifying!